src/pages/install/macos.md

```diff
@@ -0,0 +1,34 @@
+
+# Installing RGBDS on macOS
+
+RGBDS is available on [Homebrew](https://brew.sh) as the [`rgbds` package](https://formulae.brew.sh/formula/rgbds).
```

Comment:
```suggestion
RGBDS is available on [Homebrew](https://brew.sh) as the [`rgbds` package](https://formulae.brew.sh/formula/rgbds).
This may not be suitable if your version of macOS is old enough not to be supported by Brew anymore.
```

(Referencing that this involves building Rust from scratch on versions that don't get binary packages anymore.)

Reply:
"This may not be suitable if your version of macOS is too old for Brew to support it."